Document/0x05b-Basic-Security_Testing.md

```diff
@@ -756,6 +756,12 @@ An easy way to install a CA certificate is to push the certificate to the device
 
 You should then be prompted to confirm installation of the certificate (you'll also be asked to set a device PIN if you haven't already).
 
+This installs the certificate in the user certificate store. In order to place the certificate in the root store you can perform the following steps:
+1. Run adb as root with `adb root` and `adb shell`.
+2. Locate the newly installed certificate at `/data/misc/user/0/cacerts-added/`.
+3. Copy the certificate to the following folder `/system/etc/security/cacerts/`.
+4. Reboot the Android VM.
```

Comment:
This is unfortunately already outdated and will only work for Android versions below 7.0. We should put in a sentence instead saying that Network Security Configuration is enforcing that user installed CAs are not trusted by default anymore. 
There is the manual option of getting the Burp CA into the System truststore (Bypassing the Network Security Configuration) and the one we described in the MSTG in "Bypassing the Network Security Configuration"

Do you want to re-structure the paragraph instead?